beamerInitialize/index.js

```diff
@@ -17,7 +18,46 @@ function beamerInitialize(id) {
   }
 
   window.beamer_config = {
-    product_id: id
+    product_id: id,
+    selector: args.selector,
```

Comment:
as per the suggestion above we can just spread here instead of namely declaring all

Reply:
Sure! Just thought it best to restrict the params sent to beamer to only ones the API actually specifies but it's probably overkill.